src/client/admin/admin.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+package admin;
+
+import "google/protobuf/empty.proto";
+
+import "client/pfs/pfs.proto";
+import "client/pps/pps.proto";
+
+message Op17 {
```

Comment:
What do you think of using underscore to disambiguate major/minor version (i.e. `Op1_7`)? I was going to confess that it was unconventional, but I think the go linter has an explicit exception for this kind of thing: https://github.com/golang/lint/issues/47 (which, despite the skeptical comment, is fixed by https://github.com/zimmski/lint/commit/2481c997ffa02332f21d9cac2e5e2b3907a51be0)